libs/langchain/langchain/document_loaders/confluence.py

```diff
@@ -505,13 +505,18 @@ def process_page(
             ]
             text = text + "".join(comment_texts)
 
+        metadata = {
+            "title": page["title"],
+            "id": page["id"],
+            "source": self.base_url.strip("/") + page["_links"]["webui"],
+        }
+        
+        if "version" in page and "when" in page["version"]:
+            metadata["when"] = page["version"]["when"]
```

Comment:
might be better to call it more precisely: `when` -> `updated_at` / `last_modified_at`

to differentiate between update time from creation time